client/components/seo/seo-site-search-preview.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+
+export const SeoSiteSearchPreview = React.createClass( {
+	shouldComponentUpdate( next ) {
+		const current = this.props;
+
+		return ! (
+			current.snippet === next.snippet &&
+			current.title === next.title &&
+			current.url === next.url
+		);
+	},
+
+	render() {
+		const {
+			snippet,
+			title,
+			url
+		} = this.props;
+
+		return (
+			<div className="seo-search-result-preview">
```

Comment:
I don't think this adheres to our CSS naming standards (cc @mtias ). Should this file be named `index.jsx`, or directory renamed `seo-site-search-preview` ? At the very least, the class should be renamed `seo__search-result-preview`:

>The `component` fragment matches the folder name of the React component it's providing styles to.

https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines